nullaway/src/main/java/com/uber/nullaway/handlers/LibraryModelsHandler.java

```diff
@@ -304,6 +304,13 @@ private static LibraryModels loadLibraryModels() {
             .put(methodRef("java.util.ArrayDeque", "offer(E)"), 0)
             .put(methodRef("java.util.ArrayDeque", "push(E)"), 0)
             .put(methodRef("java.util.ArrayDeque", "<T>toArray(T[])"), 0)
+            .put(methodRef("java.lang.Integer", "parseInt(java.lang.String)"), 0)
```

Comment:
Can we also handle overloads that take a radix, like [this one](https://docs.oracle.com/javase/7/docs/api/java/lang/Integer.html#parseInt(java.lang.String,%20int))?  I think `Long` has one too, not sure about others

Reply:
done 👍